src/GraphQL/Buffers/EntityBuffer.php

```diff
@@ -55,6 +58,7 @@ protected function getBufferId($item) {
    */
   public function resolveBufferArray(array $buffer) {
     $type = reset($buffer)['type'];
+    $language = reset($buffer)['language'];
```

Comment:
Currently the buffer is keyed by entity type. Do we also need to key it by language as well? You are assuming that all items in this buffer want the same language. Maybe that is an ok assumption to make? Not sure here.